src/docs/concepts/timers.adoc

```diff
@@ -25,16 +25,16 @@ Timer timer = Timer
     .register(registry);
 ----
 
-NOTE: Max for basic `Timer` implementations such as `CumulativeTimer`, `StepTimer` is a time window max (`TimeWindowMax`).
+NOTE: The maximum value for basic `Timer` implementations, such as `CumulativeTimer` and `StepTimer`, is a time window maximum (`TimeWindowMax`).
```

Comment:
I worry a little this might be misconstrued as referring to the value one can record with a timer, rather than the statistical maximum of values recorded with the timer. Perhaps just changing `value` here to `statistic` is more clear. What do you think?

Reply:
I changed it to "maximum statistical value." I don't know the domain well enough to know whether that will be more clear. Let me know if it's a problem, please.